src/bin/ion/commands/generate/README.md

```diff
@@ -347,21 +343,21 @@ types defined within the outer types.
 Generated type has a `readFrom` method which can be used to read Ion data using `Reader` and initialize the generated
 type with the given Ion data. *(Generated code here depends on `ion-rs` version `1.0.0-rc.2`)*
 
-```
-let mut reader: Reader = ReaderBuilder::new().build(ION_DATA)?;
-reader.next()?;
-let foo: Foo = Foo::read_from(&mut reader)?;
+```rust
+let mut reader: Reader = ReaderBuilder::new().build(ION_DATA) ?;
+reader.next() ?;
+let foo: Foo = Foo::read_from( & mut reader) ?;
```

Comment:
Any particular reason for the added spaces here?

Reply:
huh. I removed those spaces. I think its getting formatted because I added `rust` syntax highlighting and this code block does not really contain a main. I will just remove the highlight I think.